### ISSUE TITLE
Issue with Aeon handling

### ISSUE THREAD

**Issue by keneanung** [svof/svof#100]
~~~~
Message #612        Sent by Tharvis
11/01/13:23 Found a possible issue with svof's aeon handling: even with repeated (and quickly 
consecutive) "You move sluggishly into action." messages, svof doesn't always realise there is 
actually aeon (though it's possible this is currently just limited to the Tower of the Fang).
~~~~

**Comment by vadi2** [comment]
That message without a proper aeon line is more likely to be retardation
than aeon and the system would consider it as such. A log with the affs
prompt tag would explain what's going on.

**Comment by jgh713** [comment]
Should be addressed with #209 in the future, and seemed to be limited to a single denizen's attacks without such messages.
